caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -389,18 +402,21 @@ def run_sentence_similarities(
                 Each one contains the source-sentence's score for each sentence in order.
         """
 
-        source_embedding = self._encode_with_retry(
+        source_embedding, input_token_count = self._encode_with_retry(
             source_sentences, truncate_input_tokens=truncate_input_tokens
         )
-        embeddings = self._encode_with_retry(
+        embeddings, input_token_count = self._encode_with_retry(
             sentences, truncate_input_tokens=truncate_input_tokens
         )
 
+        input_token_count = input_token_count + input_token_count
```

Comment:
The same name was used, so 2nd overwrote the first.

Reply:
fixed.